funnel/views/project.py

```diff
@@ -77,64 +77,64 @@ class CountWords:
     ),
     CountWords(
         __("Two registrations so far"),
-        __("You &amp; one other"),
+        __("You & one other"),
```

Comment:
Lazy strings should be HTML escaped, so this should be `&amp;`. The corresponding test in `tests/unit/views/project_test.py` is broken because it's still expecting `&amp;` there.